File: Algorithms/Maths/C++/Alice_Apple_Tree.cpp
#include<iostream>
using namespace std;

int main(){

    int apple, count = 0, sum = 0;
    cout << "Enter the value of the Apple: ";
    cin >> apple;

    while (sum < apple){
        count++;
        sum += (12 * count * count);
    }

    cout << "n: " << (12 * count * count) << endl;
    cout << "Sigma(" << (12 * count * count) << "): " << sum << endl;
    cout << "Count: " << count << endl;
    cout << "Perimeter: " << (8 * count) << endl;

    return 0;
}

File: Algorithms/Maths/Java/Alice_Apple_Tree.java
import java.util.Scanner;

public class Alice_Apple_Tree {
    public static void main(String[] args){

        Scanner sc = new Scanner(System.in);
        
        System.out.print("Enter the value of the Apple: ");
        int apple = sc.nextInt();

        int count = 0, sum = 0;

        while (sum < apple) {
            count++;
            sum += (12 * count * count);
        }

        System.out.println("n: " + 12 * count * count);
        System.out.println("Sigma("+12 * count * count+"): " + sum);
        System.out.println("Count: " + count);
        System.out.println("Perimeter: " + 8*count);

        sc.close();
    }
}
